interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html

```diff
@@ -16,7 +16,7 @@
   >
     {{ 'validation.upload-data.upload-result'|translate }}
   </dialogue-turn>
-  <dialogue-turn *ngIf="!uploadDataStored" [isSpoken]="noUploadDataStored">
+  <dialogue-turn *ngIf="!uploadDataStored" [isSpoken]="!noUploadDataStored">
```

Comment:
The `isSpoken`-condition needs to be the same as the `*ngIf` one..

Reply:
@RubenGeo

Reply:
I read this line like 3 times thinking the conditions are already the same